.github/CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+1. Enhancement - Added new fixes - thanks to @toms03 (Tom Earl)
```

Comment:
@cpawley might just be worth adding a little detail to this when merging. Probably...

`1. AIRAC (1906) - Added LUVUM and UMLUX fixes (Birmingham (EGBB) SIDs) - thanks to @TomS03 (Tom Earl)`